### ISSUE TITLE
Transparent fill for shields

### ISSUE THREAD

**Issue by osmpaws** [gmgeo/osmic#55]
Is there some reason, why we can not set transparent fill for a shield. Just to use shield to have eg. circle around any icon. I made a simple change in python to skip the control and set full transparency for the svg rectangle, when fill is not set in yaml config file.

**Comment by nebulon42** [comment]
Yes, I seem not to have thought about the use case with just strokes. The fix for that is to default to transparent instead of black for the shield fill. Since you already did the fix for you, you might want to do a Pull Request for that?

**Comment by osmpaws** [comment]
OK! Would you mind introducing "transparency" or better "opacity" keyword to match svg definition to shield definition in yaml? As I said I did it like when keyword fill is missing then transparency is full (0.0), otherwise icon is not transparent (1.0), but what if  we need partly transparent shield. Fill can be still set to black if it is missing and opacity can do its own part. Possibly stroke_opacity to control both fill and stroke opacity. If there is no problem I'll try to modify export.py and push it when its done.

**Comment by nebulon42** [comment]
Yes, why not. stroke_opacity and fill_opacity sound reasonable. Those should default to 1 if not set.

**Comment by osmpaws** [comment]
Well. I have done the fill_opacity and the stroke_opacity will be done ASAP.
